test/IISIntegration.FunctionalTests/IISIntegration.FunctionalTests.csproj

```diff
@@ -3,7 +3,8 @@
   <Import Project="..\..\build\common.props" />
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFrameworks>netcoreapp2.0;net461</TargetFrameworks>
```

Comment:
Do you need to cross-compile this? I thought they were all deployment based tests and you only need to run it on one TFM?